config/src/config/network_config.rs

```diff
@@ -51,6 +51,10 @@ pub const MAX_MESSAGE_SIZE: usize = 64 * 1024 * 1024; /* 64 MiB */
 pub const CONNECTION_BACKOFF_BASE: u64 = 2;
 pub const IP_BYTE_BUCKET_RATE: usize = 102400 /* 100 KiB */;
 pub const IP_BYTE_BUCKET_SIZE: usize = IP_BYTE_BUCKET_RATE;
+pub const INBOUND_TCP_RX_BUFFER_SIZE: u32 = 512 * 1024; // 3MB ~6MB/s with 500ms latency
+pub const INBOUND_TCP_TX_BUFFER_SIZE: u32 = 512 * 1024; // 1MB use a bigger spoon
```

Comment:
these two comments don't seem to match constant values, if I understand things correctly?

Reply:
need to update comments